jQueryI.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<!--<script type="text/javascript" src="jQueryI.js"></script>
+		<script type="text/javascript" src="FileStructure.js"></script>-->
```

Comment:
Good job correcting these lines. Since you no longer need them, you can remove them from your file. That will help with making your code easy to read, particularly a few weeks from now when you may have forgotten why they were there.